docs/docs/workflows/recover-coordinator.md

```diff
@@ -63,18 +63,18 @@ Assume the following `RecoveryKeys` were set in the Manifest:
 
 1. The Coordinator returned the following `RecoverySecrets`:
 
```

Comment:
Indentation is off in this listing. Two spaces for steps 1 and 2, and four spaces for step 3.
Not sure if it shows up on the webpage, but looks a bit weird in GitHub's markdown renderer